### ISSUE TITLE
[Feature Request]: Better changelogs

### ISSUE THREAD

**Issue by aaronliu0130** [ShootMe/FallGuysStats#260]
### What is your request?

"Bugfix and program optimization" is worse than no changelog. Could it be changed to things like "added new rounds" or name the specific bugs fixed?

**Comment by qutrits** [comment]
The reason I maintain this application without stopping is for my own benefit. Because despite all the time and effort put into it, no one recognizes it.

![image](https://github.com/ShootMe/FallGuysStats/assets/4685412/60fc4fd6-c9ca-4ecf-ba10-a19b40115833)

**Comment by omega32** [comment]
Great work. We all apreciate it,. It's just bad practice to keep changelogs that way. Specially when others also use the software.

At this point one have to update the app and pray that it still works as intended. And get surprised about the changed, and what broke. I mostly know because I'm working on the app (haven't committed it yet) and I see all the changes being made commit by commit.

**Comment by aaronliu0130** [comment]
Yeah! I mean, just look like how much data there is on fallalytics! Other people not contributing doesn't mean we don't like or recognize it, it merely means we don't have your brilliant mind to maintain this thing!

**Comment by aaronliu0130** [comment]
Recently, a toast on server joining was added and the overlay now displays the time of your computer.

**Comment by aaronliu0130** [comment]
I'd still appreciate it if we changed to no changelogs at all if it's only "bugfix and program optimization".

**Comment by qutrits** [comment]
I'd appreciate it if you didn't use this program.

**Comment by aaronliu0130** [comment]
What? Why?

**Comment by omega32** [comment]
At this point the project is being run very unprofessionally and unfriendly. One more example is the fact that the recent toast notifications are being handled by a DLL made by the maintainer, but it's code is not part of the project. You just need to trust it, and can't modify it. Not ideal for and open source project.

I was working on some new features, but the way this project is being handled right now completely discouraged me from contributing.

**Comment by omega32** [comment]
Sadly I don't have the time to maintain a fork, but it has crossed my mind many times.

**Comment by aaronliu0130** [comment]
I'm still unconvinced... The most unfriendly thing I've seen yet is just the single comment above though that and the dll do seem like issues.

**Comment by aaronliu0130** [comment]
Also, I doubt that you (omega32) could maintain it. C# is a far cry from JavaScript, Python or Shell.

**Comment by omega32** [comment]
I am a developer with 30+ years of experience in many languages, going all the way down to Assembler and C++, and many others... including of course the 3 languages you just mentioned. With many, many years of experience working with C#.

That's why I was working in some features I want on the tracker. But the code is constantly being changed in ways that I don't find proper. And the constant changes, breaks, lack of communications, etc. discouraged me from continuing.

**Comment by omega32** [comment]
Edited the comment up there, 30+ was an exaggeration, only 25 years actually. And still learning new thing every day.

**Comment by aaronliu0130** [comment]
Manually making changelogs, let's see how this goes :p

172 – 176:
* Formatted code and updated translations
* Add support for the FallGuy_BlueJay_UNPACKED variation
* Custom UI handling for bar and scatter plots
* Optimized creative round detail loading and handle authors' different names between different platforms
* Add "Today's Shows" button that links you to fallguys-db

179 – 182:
* Formatted code and updated translations
* Fixed profile selection and tray behavior

186 – 187:
* Add `round_robotrampage` to final rounds
* Only check ping logs if ping option is enabled
* Make bad pings display differently
* Use http to access ip2c instead of https (?)
* Auto update by default
* Fix handling for empty platform names of creative authors
* UI tweaks, code formatting and fix translations
